caching.md

```diff
@@ -64,6 +64,11 @@ TBD
 
 TBD
 
+<a name="Caching-%7B%7BOtherCommunitySupportedCacheManagers%7D%7D"></a>
+###`Other Community Supported Cache Managers`
+
+[Integration](integration.html "Integration")
```

Comment:
This works for me, but to fix the confusion about my comment about adding an anchor.

You could create a new anchor on the integration page between `Lift` and `Redis Cache Manager` called something like `cache-managers`, then change this link to:
```
[Integration](integration.html#cache-managers "Integration")
```

But, that isn't strictly needed :)